app/root.tsx

```diff
@@ -74,7 +74,7 @@ function Shell({ children }: { children: React.ReactNode }) {
       </head>
       <body className="min-h-screen w-full flex flex-col">
         <TopNav />
-        {children}
+        <main id="main">{children}</main>
```

Comment:
Just make sure we didn't use `main` already in a nested route and we are good to go!